athena-hbase/src/main/java/com/amazonaws/athena/connectors/hbase/qpt/HbaseQueryPassthrough.java

```diff
@@ -0,0 +1,88 @@
+/*-
+ * #%L
+ * athena-hbase
+ * %%
+ * Copyright (C) 2019 - 2024 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.hbase.qpt;
+
+import com.amazonaws.athena.connector.lambda.metadata.optimizations.querypassthrough.QueryPassthroughSignature;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public final class HbaseQueryPassthrough implements QueryPassthroughSignature
+{
+    // Constant value representing the name of the query.
+    public static final String NAME = "query";
+
+    // Constant value representing the domain of the query.
+    public static final String SCHEMA_NAME = "system";
+
+    // List of arguments for the query, statically initialized as it always contains the same value.
+    public static final String DATABASE = "DATABASE";
+    public static final String COLLECTION = "COLLECTION";
+    public static final String FILTER = "FILTER";
+    private static final Logger LOGGER = LoggerFactory.getLogger(HbaseQueryPassthrough.class);
+
+    @Override
+    public String getFunctionSchema()
+    {
+        return SCHEMA_NAME;
+    }
+
+    @Override
+    public String getFunctionName()
+    {
+        return NAME;
+    }
+
+    @Override
+    public List<String> getFunctionArguments()
+    {
+        return Arrays.asList(DATABASE, COLLECTION, FILTER);
+    }
+
+    @Override
+    public Logger getLogger()
+    {
+        return LOGGER;
+    }
+    /**
+     * verify that the arguments returned by the engine are the same that the connector defined
+     * @param engineQptArguments
+     * @throws IllegalArgumentException
+     */
+    @Override
+    public void verify(Map<String, String> engineQptArguments)
```

Comment:
Two things; 

1. please state why and how this differs from the original `verify` method found in the interface? (a comment stating why overriding this etc.)
2. Not calling the secondary verify method. like the parent.

Reply:
Hi @AbdulR3hman Added the comment as per your suggestion.